lib/tools/apk-utils.js

```diff
@@ -498,41 +498,50 @@ apkUtilsMethods.setDeviceLocale = async function (locale) {
  * @return {boolean} If current locale is language and country as arguments, return true.
  */
 apkUtilsMethods.ensureCurrentLocale = async function (language, country) {
-  let hasLanguage = _.isString(language);
-  let hasCountry = _.isString(country);
-
-  if (!hasLanguage && !hasCountry) {
-    log.warn('ensureCurrentLocale requires language or country');
-    return false;
-  }
-
-  const lowLanguage = (language || '').toLowerCase();
-  const lowCountry = (country || '').toLowerCase();
+  return await retryInterval(5, 1000, async () => {
+    try {
+      const hasLanguage = _.isString(language);
```

Comment:
variable setting and verification stuff might be moved out of `retryInterval` body.

Reply:
Good point. Thanks!